Packages/tests/Basic/UTF_JSONWaveNotes.ipf

```diff
@@ -184,3 +184,20 @@ static Function TestJSONWaveNoteCombinations()
 	str = GetStringFromWaveNote(wv, "string")
 	CHECK_EQUAL_STR(str1, str)
 End
+
+static Function TestWaveNoteFromJSON()
+
+	variable jsonID = JSON_PARSE("{ \"abcd\" : [1, 2]}")
+	CHECK_GE_VAR(jsonID, 0)
+
+	Make/FREE wv
+	Note/K wv, "efgh\rJSON_BEGIN\r"
```

Comment:
Maybe we should have the WAVE_NOTE_JSON_SEPARATOR non-static then to be usable in tests.